### DISCUSSION TITLE
Ability to run a query on all projects

### DISCUSSION THREAD

**Discussion by JordyZomer** [github/securitylab#83] [General] [answered]
Hello, 

It would be cool if there's the ability to run a query on all projects, this would useful when you want to fix OSS projects. I heard this  is technically possible internally, so I wondered if this could be a public feature :)

Kind Regards,

Jordy

**Comment by sj** [comment, marked as answer]
Hi Jordy! This is definitely functionality we're looking into building. At the moment, this is available on the Query Console on LGTM.com: https://lgtm.com/query. You can run your query against a large number of codebases there.

If you have a particular query in mind that you think needs running against *all* codebases for which we have CodeQL databases: get in touch with the security lab and we'll happily discuss the options with you!
